internal/cli/info.go

```diff
@@ -12,7 +12,7 @@ var ()
 var CmdInfo = &cobra.Command{
 	Use:   "info",
 	Short: "Shows information about the Epinio environment",
-	Long:  `Shows status and version for Kubernetes, Gitea, Tekton and Eirini.`,
+	Long:  `Shows status and version for Kubernetes, Tekton etc.`,
```

Comment:
I thought that we had already changed this to simply use a more generic `server components`.
```suggestion
	Long:  `Shows status and versions for epinio's server-side components.`,
```